spec/spec_helper.rb

```diff
@@ -26,6 +26,7 @@
 
 require 'dor_config'
 require 'vcr'
+require 'retries'
```

Comment:
OK, not new, but `spec_helper` later disables retry sleep for tests.  Reasonable to have here.